ui/pages/Public/components/FeatureUpdates.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { Feed, Header } from 'semantic-ui-react'
+import TextFieldView from 'shared/components/panel/view-fields/TextFieldView'
+import StateDataLoader from 'shared/components/StateDataLoader'
+
+const FeatureUpdatesFeed = ({ entries }) => {
+  const getDateFromDateStr = dateStr => (
```

Comment:
I don't think this needs to be scoped within the component as it does not rely on any of the props, you could make this a top-level helper function